pandas/tests/indexing/test_loc.py

```diff
@@ -1974,12 +1975,14 @@ def test_loc_drops_level(self):
 
 
 class TestLocSetitemWithExpansion:
-    @pytest.mark.slow
-    def test_loc_setitem_with_expansion_large_dataframe(self):
+    def test_loc_setitem_with_expansion_large_dataframe(self, monkeypatch):
         # GH#10692
-        result = DataFrame({"x": range(10**6)}, dtype="int64")
-        result.loc[len(result)] = len(result) + 1
-        expected = DataFrame({"x": range(10**6 + 1)}, dtype="int64")
+        size_cutoff = 50
+        with monkeypatch.context():
+            monkeypatch.setattr(libindex, "_SIZE_CUTOFF", size_cutoff)
```

Comment:
hmm looking at this id actually expect us to cdef `_SIZE_CUTOFF`  for a tiny perf boost.  no harm in doing this until then i guess

Reply:
As long as there's an accessible Python version to override this too, SGTM.

Reply:
i think they're mutually exclusive, but im fine with doing this for the short-to-medium term